cmd/juju/interact/pollster_test.go

```diff
@@ -504,6 +504,76 @@ Select one or more numbers separated by commas:
 	c.Check(s, jc.SameContents, []string{"one", "three"})
 }
 
+func (PollsterSuite) TestQueryArraySchemaDefault(c *gc.C) {
```

Comment:
are these missing tests for existing functionality or is this used elsewhere in the PR I missed?

Reply:
These are to test that the Default schema piece is used with the array type.  The array type was currently tested without that piece.